src/classes/AFFL_ContactAccChange_TEST.cls

```diff
@@ -40,8 +40,7 @@ public with sharing class AFFL_ContactAccChange_TEST {
     private static ID orgRecTypeID;
     private static ID householdRecTypeID;
 
-    @testSetup 
-    static void dataSetup() {
+    private static void dataSetup() {
```

Comment:
This is very interesting, @lparrott! Just curious, how were you able to figure out this is what was causing the tests to fail in parallel?

fyi, @xinnanlisf - Best practice for all unit tests. Apparently setting the custom settings in a method annotated with @testSetup causes row lock errors.

Reply:
The issue had to do with all of these methods trying to edit the same shared Contact, since tests run in parallel, they were receiving the UNABLE_TO_LOCK_ROW error.

Another way to fix this would be create a new Contact for each test, but there appears to be some setup involved, so I took the fastest approach.

This article tipped me of to the root problem: https://salesforce.stackexchange.com/questions/8058/unable-to-lock-row-error-while-running-all-apex-test-classes

Reply:
Cool. And we do have a class for creating uniquely-named Contacts, just fyi. 

I'm surprised this didn't fail sooner, but good to have it straighten now.